bridge/client/app/_views/ktb-sequence-view/ktb-sequence-view.component.ts

```diff
@@ -91,64 +91,68 @@ export class KtbSequenceViewComponent implements OnInit, OnDestroy {
   constructor(private _changeDetectorRef: ChangeDetectorRef, private dataService: DataService, private route: ActivatedRoute, public dateUtil: DateUtil, private router: Router, private location: Location) { }
 
   ngOnInit() {
-    this.currentSequence = null;
-    this.route.params
-      .pipe(takeUntil(this.unsubscribe$))
-      .subscribe(params => {
+    const projectName$ = this.route.params
```

Comment:
`projectName$` is used only once, so is it really necessary? it's difficult to read, having first the declaration of `projectName$`, then some lines of other code in between and then a one-time usage of `projectName$`

Reply:
I don't think it's harder to read. You clearly see on what `this.project$` depends on, when you assign it.